packages/web3wallet/src/types/engine.ts

```diff
@@ -61,7 +66,7 @@ export abstract class IWeb3WalletEngine {
   public abstract getPendingSessionProposals(): Record<number, ProposalTypes.Struct>;
 
   // query all pending session requests (SIGN)
-  public abstract getPendingSessionRequests(): Record<number, Web3WalletTypes.SessionRequest>;
+  public abstract getPendingSessionRequests(): PendingRequestTypes.Struct[];
```

Comment:
If we're adding to SignClient we should also expose `getPendingSessionRequests` on the SignClient itself, rather than only here.

Reply:
At least that's how I understood the decision/intention of this during the Sync call, making it simply a wrapper/proxy call here. Let me know if you're seeing it differently please 🙏

Reply:
makes sense, will update accordingly :+1: